cmd/print.go

```diff
@@ -78,12 +85,30 @@ func (pr *PrintRunner) PrintGatewaysAndHTTPRoutes(cmd *cobra.Command, _ []string
 		return err
 	}
 
-	pr.outputResult(httpRoutes, gateways)
+	var additionalResources []*unstructured.Unstructured
+	if pr.allResources {
+		additionalResources, err = i2gw.ConstructOtherResourcesFromFile(pr.namespaceFilter, pr.inputFile, pr.providers)
+		if err != nil {
+			return err
+		}
+	}
 
+	pr.outputResult(httpRoutes, gateways, additionalResources...)
 	return nil
 }
 
-func (pr *PrintRunner) outputResult(httpRoutes []gatewayv1beta1.HTTPRoute, gateways []gatewayv1beta1.Gateway) {
+func (pr *PrintRunner) outputResult(httpRoutes []gatewayv1beta1.HTTPRoute, gateways []gatewayv1beta1.Gateway, additionalResources ...*unstructured.Unstructured) {
+	for i := range additionalResources {
+		err := pr.resourcePrinter.PrintObj(additionalResources[i], os.Stdout)
+		if err != nil {
+			fmt.Printf("# Error printing %s/%s %s with: %v\n",
+				additionalResources[i].GetNamespace(),
+				additionalResources[i].GetName(),
+				additionalResources[i].GetObjectKind().GroupVersionKind().Kind,
+				err)
+		}
+	}
+
 	if len(httpRoutes)+len(gateways) == 0 {
 		msg := "No resources found"
```

Comment:
two questions; 
1) why do we print the additional resources before the gateways and http routes? Reversing the order would make more sense imo. 

2) I think "No resources found" message needs to be rephrased. Can we have a case where you have `additionalResources` but have no `Gateway` and `HTTPRoutes`?

Reply:
lets keep a warning or some other indication if the user has no "Gateway resources" (currently only referring to gateway and httproutes)

Reply:
closing thread as no more relevant after the latest changes